src/game_launcher.cpp

```diff
@@ -925,14 +926,14 @@ bool game_launcher::play_multiplayer()
 		gui2::show_error_message(video(), _("Error while playing the game: ") + e.message);
 	} catch (mapgen_exception& e) {
 		gui2::show_error_message(video(), std::string(_("Map generator error: ") + e.message));
-	} catch(network::error& e) {
+	} catch(wesnothd_error& e) {
 		if(e.message != "") {
-			ERR_NET << "caught network::error: " << e.message << std::endl;
+			ERR_NET << "caught network error: " << e.message << std::endl;
 			gui2::show_transient_message(video()
 					, ""
 					, translation::gettext(e.message.c_str()));
```

Comment:
hmm yes i thought you were refering to the strign that was changes in this line,
Afaik those errosmessages recevied by boost asio are already translated to maybe that gettext call is subflicious now